ts/WoltLabSuite/Core/Ui/Poll/Manager/Poll.ts

```diff
@@ -0,0 +1,205 @@
+/**
+ * Handles the poll UI.
+ *
+ * @author  Joshua Ruesweg
+ * @copyright  2001-2022 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Ui/Poll/Manager/Poll
+ * @since   5.5
+ */
+
+import DomChangeListener from "../../../Dom/Change/Listener";
+import DomUtil from "../../../Dom/Util";
+import { formatNumeric } from "../../../StringUtil";
+import Participants from "./View/Participants";
+import Results from "./View/Results";
+import VoteView from "./View/Vote";
+import VoteHandler from "./Vote";
+
+export enum PollViews {
+  vote = "vote",
+  results = "results",
+}
+
+export class Poll {
+  public readonly pollID: number;
```

Comment:
```suggestion
  public readonly pollId: number;
```
This is already annoying enough in PHP, please try to stick with strict camel case in TypeScript to keep it consistent.